charts/uplink/values.yaml

```
@@ -54,7 +50,7 @@ nodeSelector: {}
 tolerations: []
 affinity: {}
 postgresql:
-  url:
+  url: null
```

```suggestion
  url: ""
```
Correct empty default value.